src/cocaine-app/planner/__init__.py

```diff
@@ -55,6 +58,8 @@ def __init__(self, db, niu, job_processor, namespaces_settings):
         self.couple_defrag_timer = periodic_timer(
             seconds=self.params.get('couple_defrag', {}).get(
                 'couple_defrag_period', 60 * 15))
+        self.ttl_cleanup_timer = periodic_timer(
+            seconds=self.params.get('ttl_cleanup', {}).get('ttl_cleanup_periodic', 60 * 15))
```

Comment:
Fixed in https://github.com/yandex/mastermind/pull/311/commits/df804b5194ac65d6a365dda7cf7253cd1fc2f821